string Text
uint8 LineNumber